var/spack/repos/builtin/packages/py-ipdb/package.py

```diff
@@ -10,24 +10,20 @@ class PyIpdb(PythonPackage):
     """ipdb is the iPython debugger and has many additional features, including
     a better interactive debugging experience via colorized output."""
 
-    pypi = "ipdb/ipdb-0.10.1.tar.gz"
+    pypi = "ipdb/ipdb-0.13.11.tar.gz"
 
-    version("0.10.1", sha256="bb2948e726dbfb2687f4a582088b3f170b2556ba8e54ae1231c783c97e99ec87")
-
-    # :TODO:
-    # There might be potential to add variants here, but at the time of writing
-    # this the original packager does not know what they are. See the 3rd party
-    # section on ipdb's GitHub:
-    #     https://github.com/gotcha/ipdb#third-party-support
-    depends_on("python@2.6:2.8,3.2:")
+    version("0.13.11", sha256="c23b6736f01fd4586cc2ecbebdf79a5eb454796853e1cd8f2ed3b7b91d4a3e93")
+    version("0.13.10", sha256="6950715f491d59df6c27b49cb372f22c2f1763478a5e9ed03fb0507e2d85f460")
+    version("0.13.9", sha256="951bd9a64731c444fd907a5ce268543020086a697f6be08f7cc2c9a752a278c5")
+    version("0.13.8", sha256="8d368fa048a93ad6c1985d7f1d78d68580c879e4053fc15714bdcf2a1b042d06")
+    version("0.13.7", sha256="178c367a61c1039e44e17c56fcc4a6e7dc11b33561261382d419b6ddb4401810")
 
     # Dependencies gathered from:
     #     https://github.com/gotcha/ipdb/blob/master/setup.py
-    # However additional dependencies added below were found via testing.
     depends_on("py-setuptools", type="build")
-    # ipdb needs iPython and others available at runtime
-    depends_on("py-ipython@0.10.2:", type=("build", "link"))
-    depends_on("py-traitlets", type=("build", "link"))
-    depends_on("py-six", type=("build", "link"))
-    depends_on("py-pexpect", type=("build", "link"))
-    depends_on("py-prompt-toolkit", type=("build", "link"))
+    depends_on("py-ipython@7.17:", when="@0.13.7: ^python@3.6:", type=("build", "run"))
+    depends_on("py-toml@0.10.2:", when="@0.13.7: ^python@3.6:", type=("build", "run"))
+
+    depends_on("py-ipython@7.31.1:", type=("build", "run"))
+    depends_on("py-tomli", when="@0.13.1: ^python@3.6:3.10", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("py-ipython@7.17:", type=("build", "run"))
    depends_on("py-toml@0.10.2:", type=("build", "run"))

    depends_on("py-ipython@7.31.1:", type=("build", "run"))
    depends_on("py-tomli", when="^python@:3.10", type=("build", "run"))
```
Before ipdb 0.13.1 and before Python 3.7 aren't a thing, no need to make this so complicated